### DISCUSSION TITLE
Creating and writing into a buffer from Block

### DISCUSSION THREAD

**Discussion by obsoleszenz** [fdehau/tui-rs#702] [Q&A]
Hey, I'm trying to adapt the [popup example](https://github.com/fdehau/tui-rs/blob/master/examples/popup.rs) to be able to write some text into the body of the popup. Currently I'm trying following but the `buf.set_string()` text is not shown in the tui. Any ideas what i'm doing wrong? Thank you :)

```
let block = Block::default()
    .title("Test")
    .borders(Borders::ALL);
let area = centered_rect(60, 20, f.size());
f.render_widget(Clear, area); //this clears out the background
let mut buf = Buffer::empty(area);
buf.set_string(area.x + 2, area.y + 2, "Test234", Style::default());
f.render_widget(block, area);
```

**Comment by mindoodoo** [comment]
Hey :wave:, unfortunately this repo is no longer maintained, but pop on over to [ratatui](https://github.com/tui-rs-revival/ratatui) we're were have a community maintained fork !
